components/bookings/BookingForm.vue

```diff
@@ -237,13 +175,13 @@ import 'dayjs/locale/nl'
 const apiUrl = 'https://reserveer.dwhdelft.nl'
 
 const DINNER_TEAMS = [
-  { emoji: '🍝', name: 'Kitchen Helpers' },
-  { emoji: '⏰', name: 'In a hurry' },
-  { emoji: '🧽 ', name: 'Washing up' },
-  { emoji: '👩‍🍳', name: 'Cookies' },
-  { emoji: '🍻', name: 'Bartender' },
+  { emoji: '🍝', name: 'Kitchen Helpers', description: 'You help the chef prepare the food' },
+  { emoji: '⏰', name: 'In a hurry', description: 'You will help out another time' },
+  { emoji: '🧽 ', name: 'Washing up', description: 'You clean the dishes afterwards' },
+  { emoji: '👩‍🍳', name: 'Cookies', description: 'You are the main chef' },
+  { emoji: '🍻', name: 'Bartender', description: 'You serve the drinks' },
 ]
-const DINNER_RESTRICTIONS = ['Meat', 'Fish', 'Seafood', 'Cheese', 'Nuts', 'Dairy']
+const DINNER_RESTRICTIONS = ['Meat', 'Fish', 'Seafood', 'Cheese', 'Nuts', 'Dairy', 'Coriander']
```

Comment:
The new option kinda destroys the layout: 
<img width="611" alt="image" src="https://user-images.githubusercontent.com/15815208/211662233-b41640fb-ab3e-45ee-acd0-7713683ef03a.png">
Maybe we should remove the option or change the layout